simpleSequential/test_stringToInteger.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from stringToInteger import stringToInteger
+
+class testStringToInteger(unittest.TestCase):
+    def test_positiveNumericalStringInput(self):
+        self.assertEqual(stringToInteger("5"), 5)
+
+    def test_negativeNumericalStringInput(self):
+        self.assertEqual(stringToInteger("-6"), -6)
+
+    def test_zeroNumericalInput(self):
```

Comment:
It is often beneficial to check the whole test suite (test file) to see the standard. For example, we use "...NumericalStringInput" in the other 2 tests. So it is better for this test to follow the convention.

As I said, "zeroNumberInput" makes people think the test has no input.

Changing it to "zeroNumericalInput" makes people think the test has no numerical input, but might have something else like a float of boolean, which is not what you mean.

It is better with the name "zeroNumericalStringInput"

Reply:
This is the last commit (newest commit) of the sequence. So you can "git reset --soft HEAD~1" to remove this commit only.